test/Tests.Infrastructure/AmazonGlacierRetryTheoryAttribute.cs

```diff
@@ -37,35 +37,23 @@ static AmazonGlacierRetryTheoryAttribute()
             }
         }
 
+        public AmazonGlacierRetryTheoryAttribute()
```

Comment:
where this is used?

Reply:
I added this so in case we will want to pass it to `MultiTheoryAttribute` it will compile since it need default ctor, so `Activator.CreateInstance(t)` won't throw.